vipps-recurring-api.md

```diff
@@ -858,21 +864,21 @@ An example from a response:
 {
   "status": "FAILED",
   "type": "RECURRING",
-  "failureReason": "insufficient_funds",
-  "failureDescription": "Payment was declined by the payer bank due to lack of funds"
+  "failureReason": "user_action_required",
+  "failureDescription": "User action required"
 }
 ```
 
 Here is a list of possible values for `failureReason`, their respective descriptions and possible actions that the user/merchant could take.
 
 | Reason | Description | Action |
 | ---- | ----------- | ------ |
-| insufficient_funds | Payment was declined by the payer bank due to lack of funds. | User must either add funds to the card to cover the difference between the amount to be paid. Alternatively they can change to another, or add a new, payment source that is adequately funded to complete the transaction. |
-| invalid_card | The user tried to pay using a card that has either expired or is disabled by the issuer. | User must change, or add a new, payment source on the agreement in Vipps. |
-| verification_required | Payment declined because the issuing bank requires verification. | Ask the user to change, or add a new, payment source on their agreement in Vipps. Alternatively removing and then adding the card might solve the issue. |
-| invalid_payment_source | The provided payment source is disabled or does not exist. | User must change payment source for the agreement. |
+| user_action_required | Payment failed. Could be lack of funds, card is blocked for ecommerce, card is expired. | User will get notified in Vipps and need to take action. This could be to add funds to the card or change the card on the agreement. |
```

Comment:
maybe add something like "If you want to send an email or similar to the user, you should encourage them to open Vipps and check the payment there to see why it is not paid" ?